### PR TITLE
Xeno health and sunder are now displayed in game panel

### PR DESCRIPTION
**Pull request by ILOVEUNGA**
## About The Pull Request
Health and sunder stats added to game panel.

## Why It's Good For The Game
Plasma is already displayed in game panel, so it would be nice to have health and sunder displayed.
healthsunder

## Changelog
:cl:
qol: Health and sunder stats added to xeno gamepanel.
/:cl:

### CODE REVIEW

**Comment by Lefunny** [comment]
You are a hero

**Comment by Mechanelize** [comment]
Chad

**Comment by Sebbe9123** [comment]
Blessed

File: code/modules/mob/living/carbon/xenomorph/xenoprocs.dm [hunk 1]
```diff
@@ -222,9 +222,15 @@
 	else //Upgrade process finished or impossible
 		stat("Upgrade Progress:", "(FINISHED)")
 
+	if(xeno_caste.max_health > 0)
+		stat("Health:", "[health]/[xeno_caste.max_health]")
+
 	if(xeno_caste.plasma_max > 0)
 		stat("Plasma:", "[plasma_stored]/[xeno_caste.plasma_max]")
 
+	if(xeno_caste.max_health > 0)
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/xenoprocs.dm:231]: mistake here.

File: code/modules/mob/living/carbon/xenomorph/xenoprocs.dm [hunk 2]
```diff
@@ -222,9 +222,15 @@
 	else //Upgrade process finished or impossible
 		stat("Upgrade Progress:", "(FINISHED)")
 
+	if(xeno_caste.max_health > 0)
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/xenoprocs.dm:225]: all castes have max_health, so this is useless

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/xenoprocs.dm (hunk 2 above)
Comment (ILOVEUNGA) [reply code/modules/mob/living/carbon/xenomorph/xenoprocs.dm]: all xenos have sunder, even larva and hivemind. I dunno how to exclude them, so I'm just gonna leave it as is

**Review by BraveMole — APPROVED** [review]

**Pull request merged**